coin_app_2024-02-26.sql

```diff
@@ -0,0 +1,61 @@
+# ************************************************************
+# Sequel Ace SQL dump
+# Version 20062
+#
+# https://sequel-ace.com/
+# https://github.com/Sequel-Ace/Sequel-Ace
+#
+# Host: 127.0.0.1 (MySQL 11.2.2-MariaDB-1:11.2.2+maria~ubu2204)
+# Database: coin_app
+# Generation Time: 2024-02-26 11:34:10 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+SET NAMES utf8mb4;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE='NO_AUTO_VALUE_ON_ZERO', SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table coins
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `coins`;
+
+CREATE TABLE `coins` (
+  `Reverse` varchar(255) DEFAULT NULL,
+  `Obverse` varchar(255) DEFAULT NULL,
+  `Type` varchar(255) DEFAULT NULL,
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `Issuer` varchar(255) DEFAULT NULL,
+  `Grade` varchar(255) DEFAULT NULL,
+  `Value` decimal(11,0) DEFAULT NULL,
```

Comment:
The convention is to make all field names lower case, so change these to `reverse`, `obverse`, etc